app/assets/stylesheets/base/_base.scss

```diff
@@ -17,3 +17,10 @@
 body {
   padding: 20px;
 }
+
+.question {
+  font-weight: bold;
+  .answer label, .answer {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line